Skip update when page not found (#364)
tldr-pages/tldr-node-client: README.md

```js
{
  "repository": "http://myrepo/assets/tldr.zip"
}
```

> If you want to prevent that a cache update is performed each time no page is found for a command, then
> you can set the configuration variable `skipUpdateWhenPageNotFound` to `true` (defaults to `false`):

Reviewer: ```suggestion
By default, a cache update is performed anytime a page is not found for a command. To prevent this behavior,
you can set the configuration variable `skipUpdateWhenPageNotFound` to `true` (defaults to `false`):
```

```js
{

>   "repository" : "http://myrepo/assets/tldr.zip"

Reviewer 2: A bit out of context, but I noticed that there's a space in front of the colon. Is this correct? The other examples don'( have it.

}
```

> If you want to prevent that a cache update is performed each time no page is found for a command, then
> you can set configuration variable `skipUpdateWhenPageNotFound` to the value `true` 
> (the default value is `false`):

Reviewer 2: ```suggestion
If you want to prevent that a cache update is performed each time no page is found for a command, then
you can set the configuration variable `skipUpdateWhenPageNotFound` to `true` (defaults to `false`):
```
